### PR TITLE
Cem 2595 info header try3

### PR DESCRIPTION
**Pull request by Hunterjdf**
Basic:
image
Long:
image
Nothing: 
image
- Could add functionally (I believe you told me not to during the meeting)  if requested - otherwise will start next component.

### CODE REVIEW

**Review by ralph-dev — APPROVED** [review]
Lgtm , let's start the next component

**Comment by ralph-dev** [comment]
It would be great to add the onMouseEnter / Leave storybook actions here so you have events to hook onto.

https://upmostly.com/tutorials/react-onhover-event-handling-with-examples

**Comment by Hunterjdf** [comment]
onMouseEnter / Leave as specified by the provided documentation. 
image
Now the icon does not exist unless you mouse over it. 
That's just an example to demonstrate the onMouseEnter.
If you wanted it implemented in a different way, let me know.

**Comment by ralph-dev** [comment]
I was more talking about, OnMouseEnter, show a tooltip on the info icon :). But just emit the event don't actually show a tooltip 😅.

Interesting UX 😅 you made here 🤣

**Comment by Hunterjdf** [comment]
image
image

**Comment by Hunterjdf** [comment]
Yea that's a bit of an odd way to do it now that I look at it. 
This should be closer to what you had in mind. 
- I am doing the duo component with Russel, and so I was  more focused on trying to help him on his draggable list attempt.

File: src/Containers/InfoHeader/InfoHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    return (
+        <InfoHeaderText
+          onMouseEnter={() => setIsShown(true)}
+          onMouseLeave={() => setIsShown(false)}>
+          {text}
+          <Icon />
+          {isShown && (
+        <ToolTipText>
+          Tooltip
+        </ToolTipText>
```
Comment (ralph-dev) [inline src/Containers/InfoHeader/InfoHeader.tsx:40]: This Tooltip should be defined by the Storyfile. You can have a default tooltip but let the person using this component pass in their own tooltip / text explanations component.

Not everything will have the same explanation

**Review by ralph-dev — COMMENTED** [review]
Almost there!

**Comment by Hunterjdf** [comment]
Default: 
image
With args: 
image

**Comment by Hunterjdf** [comment]
Hello sir, I just wanted to check in and make sure that what I have is good to publish- I finished it a few days ago to work on this weeks component and was just waiting for you to confirm the last commit. 
I would like to commit it as it had been finished for 2 days now and I don't want the hassle of trying to un-merge and re-merge again so I wont commit unless you confirm that there will be no more changes.

**Review by ralph-dev — CHANGES_REQUESTED** [review]
You aren't emitting the event correctly. Let's discuss in the meeting.

**Comment by Hunterjdf** [comment]
image
- Actions example

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**